js/parser.js

```diff
@@ -17,6 +17,7 @@
 var ERMrest = (function(module) {
 
     module.ParsedFilter = ParsedFilter;
+    module.Location = Location;
```

Comment:
In my opinion, you should not need to make the `Location` constructor a member of the `module`.